pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "dominodatalab-data"
-version = "0.6.0dev1"
+version = "0.6.0dev3"
```

Comment:
We missed to bump up the version after publishing 0.6.0dev1. And just published 0.6.0dev2, so bumped up the version to 0.6.0dev3